### DISCUSSION TITLE
It would be really nice to have Korean translation.

### DISCUSSION THREAD

**Discussion by jihyojeon** [saadeghi/daisyui#876] [Ideas / feature requests]
I would like to start Korean translation, or if you think this will be too big, please let me know how to do it.

**Comment by niceplugin** [comment]
I think so too.
Is there a way to contribute as a user PR?

**Reply by NikarashiHatsu** [reply]
Of course you can do a PR

**Comment by saadeghi** [comment]
PRs are welcome!  
You can copy English translation file from here:  
https://github.com/saadeghi/daisyui/tree/master/src/docs/src/translation

And translate strings

**Comment by jihyojeon** [comment]
Great to hear, I will start it soon. Thanks.
